source/community/reactnative/src/components/inputs/input.js

```diff
@@ -236,11 +237,14 @@ export class Input extends React.Component {
 								value={this.props.value}
 							/>
 						</View>
-						<TouchableOpacity onPress={() => { this.onClickHandle(onExecuteAction, Constants.InlineAction) }}>
+						<TouchableOpacity
+							disabled={inlineAction.isEnabled == undefined ? false : !inlineAction.isEnabled} //isEnabled defaults to true
+							opacity={inlineAction.isEnabled == undefined ? 1.0 : inlineAction.isEnabled ? 1.0 : 0.5}
```

Comment:
We can remove the `undefined` check here as well.

Reply:
inlineAction inside the payload is not inherited from the BaseActionModel. Only ActionModel is inherited from the BaseActionModel. So we need to handle the null cases in the places where we consumed.